### DISCUSSION TITLE
Data Models

### DISCUSSION THREAD

**Discussion by joepperkins** [weather-gov/api#399] [Q&A]
Can you please publish your complete data model responses, json will be fine as it can be converted to any language. 

Because null value properties are not generally returned from the json API its verify difficult to generate a complete data model.

**Comment by StephenClouse** [comment]
Data models are something we plan to add to [the OpenAPI definition](https://api.weather.gov/openapi.json).

Our focus up to now has been making sure the API meets our internal goals and getting it and the infrastructure stable and reliable for operational use. Now that we are through a lot of those issues we can start filling in these gaps.

**Comment by awyork** [comment]
Any word on this?  I would love to use the openapi-code gen but with the response models missing the generated code is not useful.

**Comment by scadergit** [comment]
The development team works towrds CD, but then NCO holds releases until they have resources to collectively deploy manually. That typically happens once or twice a year. NCO has a current challenge to their schedule that could delay even further. The development branch is continually working with NCO to improve the schedule, but it does not currently compare to industry standards.
